demo-node/load.bash

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
 
-curl -L https://storage.googleapis.com/tenzir-datasets/M57/suricata.tar.zst | tar -x --zstd
-curl -L https://storage.googleapis.com/tenzir-datasets/M57/zeek.tar.zst | tar -x --zstd
+set -e
+
+curl -L https://storage.googleapis.com/tenzir-datasets/M57/suricata.json.zst | unzstd > suricata.json
```

Comment:
Why not this?

```bash
tenzir 'load https storage.googleapis.com/tenzir-datasets/M57/suricata.json.zst
       | decompress zstd
       | save stdout' > suricata.json
```

Reply:
I think we should rather get rid of the layer entirely and do the load and import within the same pipeline. Waiting for Tobi's confirmation on this.

Reply:
got verbal confirmation from Tobi